### DISCUSSION TITLE
Icons

### DISCUSSION THREAD

**Discussion by pietrop** [vidstack/player#681] [General]
Following up from [this tweet](https://twitter.com/billygozutok/status/1574270573013704704?s=20&t=0ngjzLDUOn5oXRtMf8hx-g) about the awesome icons designs, I was wondering if the icons are available via the package?

eg 
```js
import { PlayIcon } from '@vidstack/player-react';
```

**Comment by pietrop** [comment]
I found these once in the [example page](https://github.com/vidstack/player/tree/3d9ee3129089b09974110760aa17cfc6079cca90/apps/site/src/pages/index/icons)

**Comment by mihar-22** [comment]
Yep, we'll be shipping icons just like that from all of our framework packages :) They're not available yet but will be soon.

**Reply by pietrop** [reply]
Awesome, thanks! Looking forward to it! 🥁 🥳

**Comment by billygozutok** [comment]
Update: Icons are now available on NPM via `npm install media-icons`. Refer to discussion #778 for more information.

Three ways you can use Vidstack media icons today:
1. Download on NPM via `npm install media-icons`
2. Copy the [Figma](https://www.figma.com/@vidstack) community file 
3. Browse our [website](https://www.vidstack.io/media-icons) catalog

**Reply by mihar-22** [reply]
Noting here that it's also available through both our HTML and React packages:

```html
<media-icon type="play"></media-icon>
```

```js
import { PlayIcon } from '@vidstack/react'
```
